src/dataset.hpp

```diff
@@ -50,6 +50,7 @@ struct randomx_cache {
 	randomx::DatasetInitFunc* datasetInit;
 	randomx::SuperscalarProgram programs[RANDOMX_CACHE_ACCESSES];
 	std::vector<uint64_t> reciprocalCache;
+	std::string cacheData;
```

Comment:
Should be called `cacheKey` or there should be a comment clarifying that it's the key that was used for initialization (to avoid confusion with actual cache data).

Reply:
renamed